gpt_index/indices/composability/graph.py

```diff
@@ -116,13 +126,21 @@ def from_indices(
             root_index
         ]
 
+        # collect query context, e.g. vector stores
+        query_context: Dict[str, Dict[str, Any]] = {}
+        for index in list(children_indices) + [root_index]:
+            assert isinstance(index.index_struct, V2IndexStruct)
```

Comment:
raise valueerror

Reply:
The assertions are only there to help mypy. Why should they be ValueErrors?